nsv13/code/modules/munitions/ship_weapons/_ship_weapon.dm

```diff
@@ -94,6 +94,15 @@
  */
 /obj/machinery/ship_weapon/Initialize()
 	. = ..()
+	addtimer(CALLBACK(src, .proc/PostInitialize), 5 SECONDS)
```

Comment:
We can just make the proc here be the damn get_ship() after five seconds.

Reply:
Also needs to set time to maintenance